evalml/tests/automl_tests/test_automl.py

```diff
@@ -4544,3 +4544,142 @@ def test_automl_accepts_features(
         assert all(
             ["DFS Transformer" not in p for p in automl.full_rankings["parameters"][1:]]
         )
+
+
+@pytest.mark.noncore_dependency
+def test_automl_with_iterative_algorithm_puts_ts_estimators_first(
+    ts_data, AutoMLTestEnv, is_using_windows
+):
+
+    X, y = ts_data
+
+    env = AutoMLTestEnv("time series regression")
+    automl = AutoMLSearch(
+        X,
+        y,
+        "time series regression",
+        max_iterations=5,
+        problem_configuration={
+            "max_delay": 2,
+            "gap": 0,
+            "forecast_horizon": 2,
+            "time_index": "Date",
+        },
+        verbose=True,
+        automl_algorithm="iterative",
+    )
+    with env.test_context(score_return_value={automl.objective.name: 1.0}):
+        automl.search()
+
+    estimator_order = (
+        automl.full_rankings.sort_values("search_order")
+        .id.map(lambda id_: automl.get_pipeline(id_).estimator.name)
+        .tolist()
+    )
+    if is_using_windows:
+        expected_order = [
```

Comment:
This is covered because windows test pass but we don't measure coverage for windows.